src/organisation/components/pba-number-input/pba-number-input.component.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormControl, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
+import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
+import { RxReactiveFormsModule, RxwebValidators } from '@rxweb/reactive-form-validators';
+import { PbaNumberInputComponent } from './pba-number-input.component';
+
+const id: string = 'pba-number-input0';
+
+describe('PbaNumberInputComponent', () => {
+    let group: FormGroup;
+
+    let component: PbaNumberInputComponent;
+    let fixture: ComponentFixture<PbaNumberInputComponent>;
+
+    beforeEach(() => {
+        group = new FormGroup({
+            pbaNumber: new FormControl('', [
+                Validators.pattern(/(PBA\w*)/i),
+                Validators.minLength(10),
+                Validators.maxLength(10),
+                RxwebValidators.noneOf({
+                    matchValues: ['PBA0000000', 'PBA1111111']
+                }),
+                RxwebValidators.unique()
+            ])
+        });
+
+        TestBed.configureTestingModule({
+            imports: [
+                ReactiveFormsModule,
+                ExuiCommonLibModule.forChild(),
+                RxReactiveFormsModule
+            ],
+            declarations: [PbaNumberInputComponent],
+            schemas: [CUSTOM_ELEMENTS_SCHEMA],
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(PbaNumberInputComponent);
+        component = fixture.componentInstance;
+
+        component.id = id;
+        component.group = group;
+
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+    });
+
+    describe('formatPbaNumber()', () => {
+        const testCases = [
+            { input: '562', expected: 'PBA562' },
+            { input: 'pba562', expected: 'PBA562' },
+            { input: 'pbapba562', expected: 'PBA562' },
+            { input: '562pba', expected: 'PBA562' },
+            { input: '5pba62', expected: 'PBA562' },
```

Comment:
I don't think this is valid test cases, 562abcd needs to be converted to PBA562abcd, could you check how RO transform the value. In actual world, user will input the 7 alphanumeric characters without or with PBA/pba, if it's without the PBA/pba we just add the prefix.